### PR TITLE
Recommendation file for xdc7-chicago

### PR DESCRIPTION
**Pull request by xdc7**
Added a readme file for xdc7-chicago

- [x] add 1 more commit

### CODE REVIEW

**Comment by ydong8800** [comment]
this is a general comments. :+1:

File: xdc7-chicago.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+# Chicago, IL
+
+### Places to Eat
+- Giordano's
+
+### Places to Visit
+- Sear's Tower
```
Comment (ydong8800) [inline xdc7-chicago.md]: good. :laughing: joy

Comment (xdc7) [reply xdc7-chicago.md]: Thanks 👍

**Review by ydong8800 — APPROVED** [review]
good job. :100:

**Comment by xdc7** [comment]
Merging changes

**Pull request merged**